SCClassLibrary/Common/GUI/tools/ServerMeter.sc

```diff
@@ -263,4 +263,11 @@ ServerMeter {
 	isClosed {
 		^window.isClosed
 	}
+
+		position_ { |point|
```

Comment:
check the indentation here

Reply:
Thanks! Done in https://github.com/supercollider/supercollider/commit/0b0936e8cb1a4c201f4cd91d2876c6202619f953